modules/library/coverage/src/main/java/org/geotools/coverage/processing/operation/Absolute.java

```diff
@@ -79,4 +96,20 @@ protected NumberRange deriveRange(final NumberRange[] ranges, final Parameters p
         final double max = Math.abs(range.getMaximum());
         return (max<min) ? NumberRange.create(max, min) : NumberRange.create(min, max);
     }
+
+    protected void handleJAIEXTParams(ParameterBlockJAI parameters, ParameterValueGroup parameters2) {
+        if(JAIExt.isJAIExtOperation("algebric")){
```

Comment:
Lots of references to "algebric" around the code (in this class and others), should be replaced by a constant (we are using a constant reference for the operators, why not for the name of the op?)